manta-pay/src/test/compat.rs

```diff
@@ -0,0 +1,307 @@
+)> {
+    println!("Loading parameters...");
+    let mint_path = directory.join("mint.dat");
+    manta_sdk::pay::testnet::proving::Mint::download(&mint_path)?;
+    let private_transfer_path = directory.join("private-transfer.dat");
+    manta_sdk::pay::testnet::proving::PrivateTransfer::download(&private_transfer_path)?;
+    let reclaim_path = directory.join("reclaim.dat");
+    manta_sdk::pay::testnet::proving::Reclaim::download(&reclaim_path)?;
+    let proving_context = MultiProvingContext {
+        mint: ProvingContext::decode(IoReader(File::open(mint_path)?))
+            .expect("Unable to decode MINT proving context."),
+        private_transfer: ProvingContext::decode(IoReader(File::open(private_transfer_path)?))
+            .expect("Unable to decode PRIVATE_TRANSFER proving context."),
+        reclaim: ProvingContext::decode(IoReader(File::open(reclaim_path)?))
+            .expect("Unable to decode RECLAIM proving context."),
+    };
+    let verifying_context = MultiVerifyingContext {
+        mint: VerifyingContext::decode(
+            manta_sdk::pay::testnet::verifying::Mint::get().expect("Checksum did not match."),
+        )
+        .expect("Unable to decode MINT verifying context."),
+        private_transfer: VerifyingContext::decode(
+            manta_sdk::pay::testnet::verifying::PrivateTransfer::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode PRIVATE_TRANSFER verifying context."),
+        reclaim: VerifyingContext::decode(
+            manta_sdk::pay::testnet::verifying::Reclaim::get().expect("Checksum did not match."),
+        )
+        .expect("Unable to decode RECLAIM verifying context."),
+    };
+    let parameters = Parameters {
+        note_encryption_scheme: NoteEncryptionScheme::decode(
+            manta_sdk::pay::testnet::parameters::NoteEncryptionScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode NOTE_ENCRYPTION_SCHEME parameters."),
+        utxo_commitment: UtxoCommitmentScheme::decode(
+            manta_sdk::pay::testnet::parameters::UtxoCommitmentScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode UTXO_COMMITMENT_SCHEME parameters."),
+        void_number_commitment: VoidNumberCommitmentScheme::decode(
+            manta_sdk::pay::testnet::parameters::VoidNumberCommitmentScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode VOID_NUMBER_COMMITMENT_SCHEME parameters."),
+    };
+    println!("Loading parameters Done.");
+    Ok((
+        proving_context,
+        verifying_context,
+        parameters,
+        UtxoAccumulatorModel::decode(
+            manta_sdk::pay::testnet::parameters::UtxoAccumulatorModel::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode UTXO_ACCUMULATOR_MODEL."),
+    ))
+}
```

Comment:
This function may be updated once [PR#102](https://github.com/Manta-Network/manta-rs/pull/102) is merged.